packages/waku/src/lib/middleware/dev-server.ts

```diff
@@ -74,6 +69,7 @@ export const devServer: Middleware = (options) => {
   const configPromise = resolveConfig(options.config);
   const vitePromise = configPromise.then(async (config) => {
     const mergedViteConfig = await mergeUserViteConfig({
+      cacheDir: 'node_modules/.vite/waku-dev-server',
```

Comment:
Since we have multiple instances of vite, different ones might overwrite the others' cache. That's why we change it for this one.

Reply:
Can you make it an inline comment in the code?

Reply:
done